packages/misc/src/lib/misc.ts

```diff
@@ -171,7 +178,7 @@ export const logWithRequestId = (id: string, ...args: any) => {
   }
 
   // check if config is loaded yet
-  if (!globalThis?.litConfig) {
+  if (!litConfig) {
```

Comment:
Making a note to reevaluate this module of scoping of the `LitConfig` instance for the logger refactor task: https://linear.app/litprotocol/issue/LIT-3921/migrate-to-interfacing-with-logger-through-module-over-global-object